libs/block/api/src/lib/model/event.ts

```diff
@@ -0,0 +1,44 @@
+import {Field, ID, InputType, ObjectType} from '@nestjs/graphql'
+
+// Objects
+
+@ObjectType()
+export class EventBlockFilter {
+  @Field(() => [ID], {nullable: true})
+  tags?: string[]
+
+  @Field(() => [ID], {nullable: true})
+  events?: string[]
+}
+
+@ObjectType()
+export class EventBlock {
+  @Field(() => String, {nullable: true})
+  blockStyle?: string
+
+  @Field(() => EventBlockFilter)
+  filter!: EventBlockFilter
+
+  @Field(() => [Event])
+  events!: Event[]
+}
+
+// Inputs
+
+@InputType()
+export class EventBlockInputFilter {
```

Comment:
The same thing as above, could probably re-use that. 
That's also the case for a lot of the other block models.